easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.0-foss-2017b-Python-3.6.3.eb

```diff
@@ -19,12 +16,7 @@ dependencies = [
     ('Pillow', '4.3.0', versionsuffix),
 ]
```

Comment:
Isn't dask a dependency for this version?

Reply:
Same question for the rest of the diff

Reply:
Good point, fix included in 45b58458